### PR TITLE
Pharmaceuticals proofreading

### LINKED ISSUE #839: Customizing front matter with individual manuscripts
**Is your feature request related to a problem? Please describe.**
Now that individual manuscripts are being submitted and posted to arXiv, we could make the relationships between those manuscripts and the full review more clear.  Someone who reads one of the arXiv preprints may not realize the larger review exists.  Someone reading the Manubot version may want to cite one or more of the preprints because they have more traditional versioning.

**Describe the solution you'd like**
Each individual manuscript could include a statement after the title such as:
> This manuscript is one section of a larger evolving review on SARS-CoV-2 and COVID-19: https://greenelab.github.io/covid19-review/

The Manubot version of the merged review could also reference the individual preprints, and later the journal versions, at the top:
> Snapshots of individual sections are available as preprints [1, 2].

I can make the pull request with these changes but would appreciate advice on the phrasing.

### LINKED ISSUE #851: Broken references in pharmaceuticals
Commenting here in addition to gitter, the links in references 210 and 211 (Synairgen) seem to 404, can anyone confirm? They have a nonsensical headline ("people") as well.

_Originally posted by @cbrueffer in https://github.com/greenelab/covid19-review/pull/848#issuecomment-776310863_

I think the first one can be found here: https://www.globenewswire.com/news-release/2020/03/18/2002574/0/en/Synairgen-to-start-trial-of-SNG001-in-COVID-19-imminently.html

And the second here: https://www.icpcovid.com/sites/default/files/2020-08/200720-Synairgen-annmounces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf

I'm not sure what the best thing is to do for this since ideally we would reference press releases from the company's website, but they seem to be 404-ing!

### PR DESCRIPTION
**Pull request by agitter**
### Description of the proposed additions or changes

This is my proofreading of the pharmaceuticals section.  It's marked as a draft because I still plan to close #851 in this pull request and review the other references.  However, I wanted to push what I have in case it conflicts with #853.

### Related issues

Closes #839
Closes #851
#738

### Suggested reviewers (optional)

@rando2 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
AppVeyor is failing due to the Docker Hub pull rate limit.  Nothing we can do about that.

**Review by rando2 — COMMENTED** [review]
I will keep an eye out for additional updates, but here are two things I noticed! Thanks for catching all of these issues!!!

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -231,7 +229,8 @@ Despite the findings of the Solidarity trial, remdesivir remains available for t
 Follow-up studies are needed and, in many cases, are underway to further investigate remdesivir-related outcomes, with possibilities including combinations of remdesivir with other drugs such as baricitinib, which is an inhibitor of Janus kinase 1 and 2 [@doi:10.1056/NEJMoa2031994].
 
 Similarly, the extent to which the remdesivir dosing regimen could influence outcomes continues to be under consideration.
-In complement to the study that found that a 5-day course of remdesivir improved outcomes for patients with moderate COVID-19 but a 10-day course did not [@doi:10.1001/jama.2020.16349], a randomized, open-label trial compared the effect of remdesivir on 397 patients with severe COVID-19 over 5 versus 10 days [@doi:10.1056/NEJMoa2015301; @ClinicalTrials:NCT04292899].
+In complement to the study that found that a 5-day course of remdesivir improved outcomes for patients with moderate COVID-19 but a 10-day course did not [@doi:10.1001/jama.2020.16349].
+A randomized, open-label trial compared the effect of remdesivir on 397 patients with severe COVID-19 over 5 versus 10 days [@doi:10.1056/NEJMoa2015301; @ClinicalTrials:NCT04292899].
```
[the remark this answers is not in the document]
Comment (agitter) [reply content/20.pharmaceuticals.md:233]: Good catch.

**Comment by agitter** [comment]
This is ready for review.

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -370,10 +369,12 @@ This blinding seems impossible in a non-placebo trial because at the very least,
 Correction for multiple primary outcomes can be adjusted _post hoc_ by recalculating p-values, but all of the other issues were design and statistical weaknesses that cannot be corrected for.
 Additionally, the observation of drastic disparities between pre-registration and published protocol could indicate p-hacking [@doi:10.1371/journal.pbio.1002106].
 The design limitations mean that the conclusions cannot be generalized outside of the study.
+
 A second randomized trial, conducted by the Shanghai Public Health Clinical Center, analyzed whether HCQ increased rates of virological clearance at day 7 in respiratory pharyngeal swabs compared to SOC [@doi:10/drbx].
+<!-- capitalization of day is not consistent, which do we want? -->
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.pharmaceuticals.md:374]: I addressed this in #857 (I often run into trouble pulling other people's PRs locally and then trying to push them back, so this seemed safer!)

[the remark this answers is not in the document]
Comment (agitter) [reply content/20.pharmaceuticals.md]: Thanks.  I removed this comment in my branch.

**Review by rando2 — APPROVED** [review]
Screen Shot 2021-02-12 at 10 53 16 AM

Looks like it worked!

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -408,8 +408,8 @@ A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conduct
 The trial compared treatment with HCQ in conjunction with SOC to SOC alone in 150 infected patients who were assigned randomly to the two groups (75 per group).
 The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups.
 The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 >94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
-The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC vs. 21 days in SOC).
-Additionally, 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC, with the most common adverse outcome in the SOC+HCQ group being diarrhea (10% vs. 0% in the SOC group, p=0.004).
+The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC versus 21 days in SOC).
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:410]:
```suggestion
The median time to symptom alleviation was similar across different conditions (19 days in HCQ + SOC versus 21 days in SOC).
```

**Review by cbrueffer — COMMENTED** [review]
Looks great, just a few minor suggestions.

**Pull request merged**